hera_pspec/plot.py

```diff
@@ -494,3 +496,176 @@ def delay_waterfall(uvp, blpairs, spw, pol, component='real', average_blpairs=Fa
     if new_plot:
         return fig
     
+def delay_wedge(uvps, blpairs, fold=False, cosmo=True, center_line=True, horizon_lines=True, cosmo_cbar=True, suptitle=''):
+    """
+    Plot a 2D delay spectrum (or spectra) for a group of baselines.
+    
+    Parameters
+    ----------
+    uvps : UVPspec
```

Comment:
Let's make this a single `UVPSpec` or a list of `UVPSpec`, which should be concatenated inside the function.